src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -60,7 +60,7 @@ public Task StartProcessAsync()
             }
             catch (Exception ex)
             {
-                _workerProcessLogger.LogError(ex, "Failed to start Worker Channel");
+                _workerProcessLogger.LogError(ex, $"Failed to start Worker Channel. Process fileName: {_process.StartInfo.FileName}");
```

Comment:
This error will now include filename for the process.

Reply:
Is it possible to log the entire startup command (as much info as possible, maybe even working directory)?